packages/contracts/contracts/TestCollectModule.sol

```diff
@@ -1,63 +1,70 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity >=0.8.10;
+// SPDX-License-Identifier: MIT
 
-import "hardhat/console.sol";
+pragma solidity 0.8.10;
 
 import {ICollectModule} from './interfaces/ICollectModule.sol';
 import {Errors} from './libraries/Errors.sol';
 import {FeeModuleBase} from './FeeModuleBase.sol';
 import {ModuleBase} from './ModuleBase.sol';
 import {FollowValidationModuleBase} from './FollowValidationModuleBase.sol';
-
-import {EIP712} from '@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol';
 import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
-import {IERC721} from '@openzeppelin/contracts/token/ERC721/IERC721.sol';
 import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import {IERC721} from '@openzeppelin/contracts/token/ERC721/IERC721.sol';
 
 /**
  * @notice A struct containing the necessary data to execute collect actions on a publication.
  *
  * @param amount The collecting cost associated with this publication.
  * @param currency The currency associated with this publication.
- * @param collectLimit The maximum number of collects for this publication. 0 for no limit.
- * @param currentCollects The current number of collects for this publication.
  * @param recipient The recipient address associated with this publication.
  * @param referralFee The referral fee associated with this publication.
- * @param followerOnly True if only followers of publisher may collect the post.
- * @param endTimestamp The end timestamp after which collecting is impossible. 0 for no expiry.
+ * @param followerOnly Whether only followers should be able to collect.
  */
 struct ProfilePublicationData {
     uint256 amount;
     address currency;
-    uint96 collectLimit;
-    uint96 currentCollects;
     address recipient;
     uint16 referralFee;
     bool followerOnly;
-    uint72 endTimestamp;
 }
 
-contract QuadraticFundingCollectModule is FeeModuleBase, FollowValidationModuleBase, ICollectModule {
+/**
+ * @title FeeCollectModule
+ * @author Lens Protocol
+ *
+ * @notice This is a simple Lens CollectModule implementation, inheriting from the ICollectModule interface and
+ * the FeeCollectModuleBase abstract contract.
+ *
+ * This module works by allowing unlimited collects for a publication at a given price.
+ */
+contract TestCollectModule is FeeModuleBase, FollowValidationModuleBase, ICollectModule {
```

Comment:
Not sure, what is the purpose of this Module? If it's for testing the module you're building, you should test on the module you're building. If it's to expose some methods or logging for some reason, it should simply inherit the module you're building.

Reply:
deleted it.  it was left over something I was trying.  shouldn't have been in the pr.